pkg/cosign/tuf/client.go

```diff
@@ -247,7 +258,7 @@ func GetTarget(ctx context.Context, name string, out client.Destination) error {
 
 	// Retrieves the target and writes to out. This may make a network call and cache if
 	// the embedded or cached root is invalid (e.g. expired).
-	return getTargetHelper(name, out, c)
+	return getTargetHelper(name, out, c, false)
```

Comment:
I don't understand why this path doesn't require coherence, is there someone else we can tag in?

cc @dlorenc

Reply:
This helper is doing a bit too much, IMHO. 

In general we shouldn't be re-syncing with the remote TUF repo unless absolutely necessarily, and certainly not on every single read of the local cache

Reply:
I'd grab @asraa

Reply:
I'd make this `false /* requiresCoherence */` and add a comment explaining why this path doesn't require it.

Reply:
I'm doing some clean-up here, just FYI. We should be requiring coherence between the target we have the metadata we have locally. That's what this does here. We can disable the remote check, but we should issue a warning that the current `targets.json` and the `target` you have don't line up (this probably only happens if you have a stale cache, or something else wonky happened).